pycomlink/processing/wet_dry/nearby_wetdry.py

```diff
@@ -241,26 +199,28 @@ def nearby_wetdry(
             # calculate the F values which can be used as outlier filter
             F_val = ds_nearby_cmls.deltaPL.sel(cml_id=cmlid) - medianPL
             F.loc[dict(cml_id=cmlid)] = F_val.rolling(time=timeperiod * 4).sum(
-                skipna=True) * (interval / 60)
+                skipna=True
+            ) * (interval / 60)
 
             wet_tmp = wet.copy()
 
             # if wet is true and deltaP < -2db then set two timesteps before and
             # one after to wet
             for shift in [1, -1, -2]:
                 wet.loc[dict(cml_id=cmlid.values)] = xr.where(
-                    ((wet_tmp.loc[dict(cml_id=cmlid)] == 1) & (
-                            ds_nearby_cmls.sel(
-                                cml_id=cmlid).deltaP < -2)).shift(
-                        time=shift),  ##
-                    # shift here
+                    (
+                        (wet_tmp.loc[dict(cml_id=cmlid)] == 1)
+                        & (ds_nearby_cmls.sel(cml_id=cmlid).deltaP < -2)
+                    ).shift(
+                        time=shift  # shift here
+                    ),
                     x=1,
                     y=wet.loc[dict(cml_id=cmlid.values)],
                 )
 
             wet.loc[dict(cml_id=cmlid.values)] = xr.where(
```

Comment:
I suggest to add a comment here that the following lines of code are required to reset all wet values that should remain untouched, since the shift-for-loop will create NaNs (if I understand correctly...) at the edge of the shifted wet events.

Reply:
Anyway, the code here is pretty obfuscated, but I also cannot come up with a simpler solution to pad the `wet` periods to the left and right. Hence, I suggest to leave it like that, but document clearer what is done.

Reply:
The second `xr.where()` (line 261/221) in the snippet above is necessary to keep nans at all time steps where the shift overwrote them. I added several inline comments to make the two `xr.where()` calls better understandable.